apps/web/public/components/icon/demo/stroke-width.md

```
@@ -0,0 +1,35 @@
+      </div>
+
+      <div class="flex flex-col items-center gap-2">
+        <z-icon zType="house" [zStrokeWidth]="4" />
+        <span class="text-xs text-muted-foreground">Stroke 4</span>
+      </div>
+    </div>
+  `,
+})
+export class ZardDemoIconStrokeWidthComponent {}
+
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)